test/Directory.Build.props

```diff
@@ -26,6 +26,7 @@
   </PropertyGroup>
   
   <ItemGroup>
+    <PackageReference Include="Microsoft.DotNet.PlatformAbstractions" Version="2.1.0" />
```

Comment:
Let's not use this.  And for sure every project doesn't need this reference, even if we were going to use it in one or two projects.  `.props` files are only for things that are common across many/all projects.